cloud/ovirt/ovirt_vms.py

```diff
@@ -0,0 +1,583 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+#
+# Copyright (c) 2016 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+try:
+    import ovirtsdk4 as sdk
+    import ovirtsdk4.types as otypes
+except ImportError:
+    pass
+
+from ansible.module_utils.ovirt import *
+
+
+DOCUMENTATION = '''
+---
+module: ovirt_vms
+short_description: "Module to manage Virtual Machines in oVirt."
+version_added: "2.2"
+author: "Ondra Machacek (@machacekondra)"
+description:
+    - "Module to manage Virtual Machines in oVirt."
+options:
+    name:
+        description:
+            - "Name of the the Virtual Machine to manage. Either C(id) or C(name) is required."
+    id:
+        description:
+            - "ID of the the Virtual Machine to manage. Either C(id) or C(name) is required."
+    state:
+        description:
+            - "Should the Virtual Machine be running/stopped/present/absent/suspended."
+            - "I(present) and I(running) are equal states."
+            - "C(Note:)"
+            - "For states I(absent), I(suspended) only C(name) and C(state) parameters are required.
+               All other parameters are used for I(present), I(running) and I(stopped) states"
+        choices: ['running', 'stopped', 'present', 'absent', 'suspended']
+        default: present
+    cluster:
+        description:
+            - "Name of the cluster, where Virtual Machine should be created. Required if creating VM."
+    template:
+        description:
+            - "Name of the template, which should be used to create Virtual Machine. Required if creating VM."
+            - "If template is not specified and VM don't exists, VM will be created from I(Blank) template."
+    memory:
+        description:
+            - "Amount of memory of the Virtual Machine. Prefix uses IEC 60027-2 standard."
```

Comment:
What is IEC 60027-2? Simply state if it's in MB/GB/TB or MiB/GiB/TiB (I hope the former).